packaging/format-docs.py

```diff
@@ -81,14 +93,21 @@ def format_docs(version, collectionName, types):
                         elif 'Require' in attributes:
                             defaultValue = '*(required)*'
 
-                        print(f'| {prop["PropertyName"]} | {defaultValue} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {defaultValue} | {typeName} | {prop["Description"]} |')
                     else:
-                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {typeName} | {prop["Description"]} |')
+
+    if len(relatedEnums) > 0:
+        print('\n# Related enums:\n')
```

Comment:
```suggestion
        print('\n## Possible Values\n')
```

maybe? To avoid programmer speak.

Reply:
I will switch this to "Related value types:" to catch enums and potentially future complex types if we ever agree on my POCO types for traitInfo fields idea.
Also there is a suggestion below that I add "Possible values:" in front of the enum's values, so those two wouldn't go well together.